dashboard/app/entities.go

```diff
@@ -120,10 +120,38 @@ type Bug struct {
 	DailyStats     []BugDailyStats
 	Tags           BugTags
 	DiscussionInfo []BugDiscussionInfo
+	TreeTests      BugTreeTestInfo
+}
+
+type BugTreeTestInfo struct {
+	// NeedPoll is set to true if this bug needs to be considered ASAP.
+	NeedPoll bool
+	// NextPoll can be used to delay the next inspection of the bug.
+	NextPoll time.Time
+	// List contains latest data about cross-tree patch tests.
+	List []BugTreeTest
+}
+
+type BugTreeTest struct {
+	CrashID int64
```

Comment:
If we are storing crash ID here, shouldn't we add a CrashReference so that it's not garbage collected?

Reply:
The selected crash will get references indirectly via the `addTestJob` call (and `BugTreeTest` records will only be saved if we are to save a job).